pkg/monitoring.go

```diff
@@ -591,8 +630,9 @@ func MonitorAttestationsAndProposals(ctx context.Context, s *prysmgrpc.Service,
 		registry.MustRegister(epochGauge, epochMissedProposalsGauge, epochCanonicalProposalsGauge,
 			epochOrphanedProposalsGauge, epochMissedAttestationsGauge, lastEpochMissedAttestationsGauge,
 			epochCanonicalAttestationsGauge, epochOrphanedAttestationsGauge, epochDelayedAttestationsOverToleranceGauge,
+			lastProposedEmptyBlockSlotGauge,
 			totalMissedProposalsCounter, totalCanonicalProposalsCounter, totalOrphanedProposalsCounter,
-			totalMissedAttestationsCounter, totalCanonicalAttestationsCounter, totalOrphanedAttestationsCounter,
```

Comment:
Is removing `totalCanonicalAttestationsCounter` expected?

Reply:
Good catch!  Corected